code/game/objects/structures/pool.dm

```diff
@@ -183,7 +184,7 @@
 												"<span class='userdanger'>You fall in the drained pool, and crack your skull!</span>")
 					var/obj/item/organ/limb/O = H.get_organ("head") //NT makes the most dangerous pools : they aim for the head.
 					H.apply_damage(15, BRUTE, O)
-					H.Weaken(10) // This should hurt. And it does.
+					H.Paralyse(10) // You literally cracked your fuckin skull, ofcourse you'll be unconscious
 					H.adjustBrainLoss(30) //herp
 					H.swimming = 1
```

Comment:
Did you even see where this commento it's placed you fuck,i meant this H.swimming = 1